### PR TITLE
Easy fixes

### LINKED ISSUE #866: Document USB serial setup for multicontainer with /dev/ttyACM0
Front logo [Front conversations](https://app.frontapp.com/open/top_d45n)

### LINKED ISSUE #914: Document how to  rename Applications

### LINKED ISSUE #980: Device URL documentation should list supported protocols
We should make explicit which protocols and layers of traffic are supported via the [public device URL endpoints](https://www.balena.io/docs/learn/manage/actions/#enable-public-device-url).

Front logo [Front conversations](https://app.frontapp.com/open/top_fltv)

### LINKED ISSUE #982: Pages that use restricted supervisor env vars should point to the labels conf as well
For example pages like the following, demonstrate how to use the `BALENA_SUPERVISOR_ADDRESS` but don't mention at all that the use needs to set the `io.balena.features.supervisor-api` label and `network_mode` for the service.
* https://www.balena.io/docs/learn/develop/runtime/
* https://www.balena.io/docs/reference/supervisor/supervisor-api/

Such pages should have a reference to https://www.balena.io/docs/learn/develop/multicontainer/#labels as well so that users do not miss that and reach to support.

Front logo [Front conversations](https://app.frontapp.com/open/top_frbv)

### LINKED ISSUE #983: `io.balena.features.supervisor-api` mentions that it needs `network_mode` to be also configured
> Ensures that BALENA_SUPERVISOR_HOST, BALENA_SUPERVISOR_PORT, BALENA_SUPERVISOR_ADDRESS, and BALENA_SUPERVISOR_API_KEY are added to the container environment variables, so the supervisor API can be used. (Currently will only work for services that have network_mode = host or bridge )

This caused some confusion and made it difficult to get the containers to talk w/ each other.
In my tests this wasn't needed on a Resin OS 2.12.7+rev1 device. We might need to update that section.

See: https://www.balena.io/docs/learn/develop/multicontainer/#labels

Front logo [Front conversations](https://app.frontapp.com/open/top_frdv)

### LINKED ISSUE #990: User confused by docker compose label docs
In at least on case, even though the user was pointed to and read the relevant section at https://www.balena.io/docs/reference/supervisor/docker-compose/#labels , they used

```
labels:
  - io.balena.features.kernel-modules
```

instead of the correct

```
labels:
  - "io.balena.features.kernel-modules=1"
```

or
```
labels:
  io.balena.features.kernel-modules: '1'
```

syntax in their docker compose.

It might worth improving the docs with an example.

Front logo [Front conversations](https://app.frontapp.com/open/top_fysz)

### LINKED ISSUE #1012: Docs should clarify that old OS versions still need io.resin labels
Similarly to how we've clarified the RESIN_ env vars in the supervisor docs https://github.com/balena-io/balena-supervisor/blob/master/docs/API.md#http-api-reference

Front logo [Front conversations](https://app.frontapp.com/open/top_gr3v)

### LINKED ISSUE #1046: Balena fin link broken in getting started.
User report: I just noticed that the balenaFin link on https://www.balena.io/docs/learn/getting-started/fincm3/python/ leads to https://store.resin.io/collections/balena-fin-home-page/products/fin-8gb which gives a 404.

I think we should scan the entire docs website for broken links and fix them in one go.

### PR DESCRIPTION
**Pull request by shaunmulligan**
This PR fixes a bunch of small/easy issues, I tried to do an issue per commit:
- closes #1046
- change Table 91. gsm setting to Table 77. gsm setting in shared/network/2.x.md
- closes #1012
- closes #990
- closes #983
- closes #982
- closes #980
- closes #914
- closes #866

### CODE REVIEW

File: shared/general/labels.md [hunk 1]
```diff
@@ -1,13 +1,28 @@
+__Note:__ If you have devices in your app that have a supervisor version lower than 7.22.0, then you should use the `io.resin.features.` form of the labels to ensure that all devices obey the label. Earlier supervisor version will not understand the `io.balena.features` label.
```
Comment (xginn8) [inline shared/general/labels.md]: microscopic nit:
```suggestion
__Note:__ If you have devices in your app that have a supervisor version lower than 7.22.0, then you should use the `io.resin.features.` form of the labels to ensure that all devices obey the label. Earlier supervisor versions will not understand the `io.balena.features` label.
```

File: pages/learn/manage/actions.md [hunk 2]
```diff
@@ -90,10 +90,26 @@ The `Delete Device` action is an extremely dangerous action and results in disas
...
 
+These actions can be found on the "Actions" menu for each application and apply to the application and all the devices in the fleet.
+
 ### Change Application Type
 
 This option allows you to convert your application to [another type][app-types], as long as the devices in the application meet the {{ $names.os.lower }} version requirements and your account has the appropriate privileges.
 
+### Rename Application
+
+This action allows you to rename your application. This action is only available for new [applications types][app-types] such as Starter, Microservices or Essentials. Its not currently possible to rename `Legacy` or `Classic` applications, you will first need to upgrade your app type.
```
Comment (xginn8) [inline pages/learn/manage/actions.md]: tiny nit for consistency:

```suggestion
This action allows you to rename your application. This action is only available for new [applications types][app-types] such as `Starter`, `Microservices` or `Essentials`. Its not currently possible to rename `Legacy` or `Classic` applications, you will first need to upgrade your app type.
```

File: shared/general/labels.md (hunk 1 above)
Comment (shaunmulligan) [reply shared/general/labels.md]: fixed in cb44b287684d0fed3483305b860cf31e2b55f015

File: pages/learn/manage/actions.md (hunk 2 above)
Comment (shaunmulligan) [reply pages/learn/manage/actions.md]: fixed in cb44b287684d0fed3483305b860cf31e2b55f015

**Pull request merged**